integration-tests/tests/src/tests/sync/app.ts

```diff
@@ -108,9 +108,10 @@ describe("App", () => {
       const credentials = Realm.Credentials.anonymous();
       await expect(app.logIn(credentials)).to.be.rejectedWith(
         select({
-          reactNative: "Network request failed: Could not connect to the server",
-          default:
+          reactNative: new RegExp("Network request failed.*connect"),
+          default: new RegExp(
```

Comment:
Why are there not simple strings? If they must be regexp consider using the `/ /` syntax.

Reply:
The reactNative one must be a regex. I skimmed through the documentation and thought that the `new RegExp` was the easiest readable declaration, but this is just based on my flavour so I can go with `/ /` as well

Reply:
Why does it need to be regex? Could it not just be `"Network request failed"`?

Reply:
yeah fair, thought it was too generic at first, but in hindsight, my regex doesn't provide much more detailed information